frontend/src/container/Home.jsx

```diff
@@ -53,7 +55,9 @@ export class Home extends React.Component {
     ])
 
     props.registerLiveMessageHandlerList([
-      { entityType: TLM_ET.USER, coreEntityType: TLM_CET.MODIFIED, handler: this.handleUserModified }
+      { entityType: TLM_ET.USER, coreEntityType: TLM_CET.MODIFIED, handler: this.handleUserModified },
+      { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.CREATED, handler: this.handleWorkspaceMemberCreated },
+      { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.DELETED, handler: this.handleWorkspaceMemberDeleted }
```

Comment:
I don't think these handler will affect the Home.jsx component.
They will affect the sidebar though
In that case, I think you should let the Sidebar.jsx component handle them

Reply:
When the user has no workspace and someone adds it to some workspace, the Home screen changes and the opposite is also valid. If he only has one workspace and someone deletes it he should render HomeNoWorkspace instead of HomeHasWorkspace.